cmd/minishift/cmd/start.go

```diff
@@ -108,7 +109,12 @@ For the latter see 'minishift config -h'.`,
 		Run: runStart,
 	}
 
+	// enable experimental flags
+	//startCmd.Flags().BoolVar(&future, "future", false, "Enable experimental features (unsupported)")
```

Comment:
Fair enough. For what it's worth, I still believe that what we try to do is possible if one use pflag and Viper in combination. I am fine though with doing the env variable based approach so.